tests/App/config/config.yaml

```diff
@@ -4,7 +4,7 @@ imports:
 
 framework:
     test: true
-    secret: secret
+    secret: '50n474.U53r'
```

Comment:
This was intended to be "SONATA.USER", because this secret was added in SonataUserBundle.

Reply:
IMHO it will be nice to have this secret in sonata test